tv.kodi.Kodi.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk9"
```

Comment:
Is there a specific reason for JDK9, given that it's out of software support? JDK10 and JDK11 are now available.

Reply:
I can find 10 on flathub so I will try that version